src/factories/GGVStrategyFactory.sol

```diff
@@ -8,16 +8,24 @@ import {StrategyCallForwarder} from "src/strategy/StrategyCallForwarder.sol";
 contract GGVStrategyFactory is IStrategyFactory {
     address public immutable TELLER;
     address public immutable BORING_QUEUE;
+    address public immutable STETH;
+    address public immutable WSTETH;
 
-    constructor(address _teller, address _boringQueue) {
+    constructor(address _teller, address _boringQueue, address _steth, address _wsteth) {
```

Comment:
there is a change in strategy. steth and wsteth will be pulled from Pool cotnract